src/index.html

```diff
@@ -7,7 +7,47 @@
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body >
+    <main class="card" data-qa="card">
+
+      <img
+      src="../src/images/imac.jpeg"
+      alt="main picture"
+      class="card__image">
+
+
+      <section class="card__discription">
+        APPLE A1419 iMac 27" Retina<br>5K Monoblock (MNED2UA/A)
+      </section>
+
+      <section class="card__prodact">
+        Product code: 195434
+      </section>
+
+      <section class="card__review">
+        <div class="card__review-stars">
+          <div class="card__review-stars-star"></div>
+          <div class="card__review-stars-star"></div>
+          <div class="card__review-stars-star"></div>
+          <div class="card__review-stars-star"></div>
+          <div class="card__review-stars-star"></div>
+        </div>
+        <div class="card__review-count">Reviews: 5</div>
```

Comment:
Need to use the semantic tag for text everywhere

Reply:
<img width="621" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/365ad717-74eb-416e-ad68-0c00c4c1180a">
span or p

Reply:
Still not fixed